backend/src/models/student.js

```diff
@@ -0,0 +1,75 @@
+const { Model } = require('objection')
+const tableStudents = 'students'
+
+class Student extends Model {
+  static get tableName () {
+    return tableStudents
+  }
+
+  static get idColumn () {
+    return ['StudentID']
+  }
+
+  fullName () {
+    return `${this.firstName} ${this.lastName}`
+  }
+
+  static get relationMappings () {
+    const NativeLanguage = require('./nativeLanguage')
+    const Status = require('./status')
+    const StatusUpdate = require('./statusUpdate')
+
+    return {
+      nativeLanguage: {
+        relation: Model.HasOneRelation,
+        modelClass: NativeLanguage,
+        join: {
+          from: 'students.NativeLanguageID',
+          to: 'nativeLanguages.NativeLanguageID'
+        }
+      },
+
+      statusID: {
+        relation: Model.HasOneRelation,
+        modelClass: Status,
+        join: {
+          from: 'students.StatusID',
+          to: 'statuses.StatusID'
+        }
+      },
+
+      statusUpdateID: {
+        relation: Model.HasManyRelation,
+        modelClass: StatusUpdate,
+        join: {
+          from: 'students.StudentID',
+          to: 'statusUpdates.StudentID'
+        }
+      }
+    }
+  }
+
```

Comment:
I suggest we add a `previousStatus` relation too:

```suggestion
      previousStatus: {
        relation: Model.HasOneThroughRelation,
        modelClass: Status,
        join: {
          from: 'students.StudentID',
          through: {
            from: 'statusUpdates.StudentID',
            to: 'statusUpdates.PreviousStatusID'
          },
          to: 'statuses.StatusID'
        }
      }
```